src/core/Colors/Colors.tsx

```diff
@@ -32,33 +32,36 @@ export class Colors extends Component<ColorsProps> {
   render() {
     const { colors } = this.props;
     const props = withSuomifiDefaultProps(this.props);
+    const useColors = !!colors ? colors : props.tokens.colors;
     return (
       <ColorsContainer>
-        {Object.entries(!!colors ? colors : props.tokens.colors).reduce<
-          JSX.Element[]
-        >((arr, [key, value]) => {
-          const hslaAsHex = hslaToHex(value.toString());
-          const item = (
-            <Color
-              keyName={key.toString()}
-              color={value.toString()}
-              key={key.toString()}
-              onClick={copyKey(key.toString())}
-              {...props}
-            >
-              <div className="fi-color_name">{value.toString()}</div>
-              {!!hslaAsHex && (
-                <div className="fi-color_name fi-color_name--hex">
-                  {hslaAsHex}
+        {Object.entries(useColors).reduce<JSX.Element[]>(
+          (arr, [key, value]) => {
+            const color = value.hsl;
+            const hslaAsHex = hslaToHex(color.toString());
+            const item = (
+              <Color
+                keyName={key.toString()}
+                color={color.toString()}
+                key={key.toString()}
+                onClick={copyKey(key.toString())}
+                {...props}
+              >
+                <div className="fi-color__name">{color.toString()}</div>
```

Comment:
What is the value type of color if not string? ```value.hsl``` from tokens should already be of type string?